nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -152,6 +154,33 @@ NRF_SO_EXCEPTIONAL_DATA
 
    The socket option is supported from modem firmware v2.0.0.
 
+NRF_SO_KEEPOPEN
+   Keep the socket from forcibly being closed upon PDN disconnection or reactivation events, or when the device is put into flight mode.
```

Comment:
```suggestion
   Keep the socket from being closed upon PDN disconnection or reactivation events, or when the device is put into flight mode.
```

Reply:
Do we need to separate flight mode?

Reply:
I think we should mention that `CFUN=4` let the socket remain open (if possible).

Reply:
```suggestion
   Keep the socket from forcibly being closed upon PDN disconnection or reactivation events, or when the device is put into flight mode (CFUN=4).
```